trinkets.py

```diff
@@ -1188,4 +1191,34 @@ def activate(self, time, player, sim):
             'cooldown': 120,
         },
     },
+    'Deaths Verdict heroic': {
+        'type': 'proc',
+        'passive_stats': {
+            'attack_power': 288,
+        },
+        'active_stats': {
+            'stat_name': 'Agility',
+            'stat_increment': 510,
+            'proc_name': 'Darkmoon Card: Greatness',
```

Comment:
would make the proc_name field here either "Death's Verdict" or whatever the actual name of the proc buff is, helps with making the sim logs clearer